### PR TITLE
Add primes counting function

### PR DESCRIPTION
**Pull request by yosra99**

### CODE REVIEW

**Comment by OmarHatem28** [comment]
if the input is string, it won't be a string of integers, so u cant change it and put it in an integer array.
if the user chooses to input string he will enter some characters, in the case of your function, i believe that you should just print the array as is.

File: src/com/company/Main.java [hunk 1]
```diff
@@ -74,17 +76,56 @@ private static void printMenu() {
...
+    /**
+     * Counts the number of primes in the global array arr.
+     */
+    private static void countPrime() {
+        System.out.println("Primes Count");
+
+        boolean prime = true;
+        int count = 0;
+
+        int[] numbers = arr;
```
Comment (mhmdsamir92) [inline src/com/company/Main.java]: Here you are doing a shallow copy which is ok for now but when execute all is executed the input array may change during the execution of any function
try execute all now to see what I mean

Comment (mhmdsamir92) [inline src/com/company/Main.java]: Here you are doing a shallow copy which is ok for now but when execute all is executed the input array may change during the execution of any function
try execute all now to see what I mean

Comment (OmarHatem28) [reply src/com/company/Main.java]: don't worry, I told them that anyone who will need to edit the array, should make a copy of it first then edit the copy.
also i wont approve any PR that violates this rule.

Comment (yosra99) [reply src/com/company/Main.java]: I just did not notice that I used the array resulting from the previous function in my function in "execute all".
I used to think that each function works on the original array.
I fixed it.Thanks.

**Review by OmarHatem28 — APPROVED** [review]

**Review by LinaSabboula — APPROVED** [review]

**Pull request merged**